Lib/Configurations/Release.xcconfig

```diff
@@ -5,5 +5,5 @@ DEBUG_INFORMATION_FORMAT = "dwarf-with-dsym";
 ENABLE_NS_ASSERTIONS = NO;
 MTL_ENABLE_DEBUG_INFO = NO;
 VALIDATE_PRODUCT = YES;
-SWIFT_OPTIMIZATION_LEVEL = -Owholemodule;
+SWIFT_OPTIMIZATION_LEVEL = -Onone;
```

Comment:
@colemancda I didn't reproduce that. Does it occur at build time, or runtime? I confirmed it builds successfully in Release configuration, and pass all tests. I think it isn't necessary.